apps/wallet-mobile/src/features/SetupWallet/useCases/SelectWalletFromList/SelectWalletFromListScreen.tsx

```diff
@@ -52,6 +52,7 @@ export const SelectWalletFromList = () => {
     onSuccess: ([wallet, walletMeta]) => {
       selectWalletMeta(walletMeta)
       selectWallet(wallet)
+      walletManager.selectedWalletId = walletMeta.id
```

Comment:
from now one, where there is a wallet select set, should also update the manager (will be refactor with walletManager)

Reply:
I'm not big fan of classes in js, suggestion accepted, migrating to a function.